pandas/tests/io/test_feather.py

```diff
@@ -113,10 +113,11 @@ def test_read_columns(self):
         columns = ["col1", "col3"]
         self.check_round_trip(df, expected=df[columns], columns=columns)
 
-    def read_columns_different_order(self):
+    def test_read_columns_different_order(self):
```

Comment:
@alimcmaster1 this test was never running because its name didn't start with `test_` (just FYI, no blame!)